docs/rule/gen_aidl.soy

```diff
@@ -39,7 +39,8 @@ from <code>.aidl</code> files.
   {param desc}
   The search path for import statements for the aidl command. 
   (This is the <code>-I</code> argument when invoking aidl 
-  from the command line.)
+  from the command line. It should be the base dir where all aidl files are, 
```

Comment:
This really depends on your code structure, right? You can say that "For many apps it will be the base dir..." but avoid saying "it should be" as that implies that either buck or the aidl tool care/enforce this, which I don't think is true.